model-api/src/commonMain/kotlin/org/modelix/model/data/MetaModelData.kt

```diff
@@ -43,7 +43,12 @@ data class ConceptData(
     val references: List<ReferenceLinkData> = emptyList(),
     val extends: List<String> = emptyList(),
     override val deprecationMessage: String? = null,
-) : IDeprecatable
+    val metaProperties: MutableMap<String, String> = mutableMapOf(),
+) : IDeprecatable {
+    companion object {
+        const val ALIAS_KEY = "alias"
```

Comment:
It's used in the metamodel-export MPS solution.
Should I keep it like this or just hardcode the string into the solution, since it's not used anywhere else right now?